pandas/tests/frame/methods/test_count.py

```diff
@@ -16,17 +16,22 @@ def test_count_multiindex(self, multiindex_dataframe_random_data):
         frame = frame.copy()
         frame.index.names = ["a", "b"]
 
-        result = frame.count(level="b")
-        expected = frame.count(level=1)
+        with tm.assert_produces_warning(FutureWarning):
```

Comment:
can you move these to a new test_count_with_level_deprecated

Reply:
yep, done